contracts/ColonyStorage.sol

```diff
@@ -85,10 +85,18 @@ contract ColonyStorage is DSAuth, DSMath {
 
   uint8 constant OWNER_ROLE = 0;
   uint8 constant ADMIN_ROLE = 1;
+  uint8 constant RECOVERY_ROLE = 2;
   uint8 constant MANAGER = 0;
   uint8 constant EVALUATOR = 1;
   uint8 constant WORKER = 2;
 
+  // Variables for recovery mode
+  bool recoveryMode;
+  uint64 recoveryRolesCount;
```

Comment:
Why `uint64`? I think it's just more expensive than `uint256` here, no?

Reply:
I am being outvoted on whether we should be optimising for recovery mode or the Magical Christmasland where we don't need to use it, so this can stand for now 😄

Reply:
✨ 🎄 ⛰ 🚫